### ISSUE TITLE
still topics to follow shown

### ISSUE THREAD

**Issue by HikaruIchijyo** [Dr-TSNG/TwiFucker#44]
even though option to dosable that is already enabled i am using latest lspatched apk file sent on tg channel which u uploaded today

oh and mostly in 2nd account havent found any in 1st account yet though

**Comment by HikaruIchijyo** [comment]
this also possible to remove
promoted content
& viral tweets
viral pictures
viral videos etc

**Comment by aliafshany** [comment]
i get them too
i installed many twitters clients to solve it, but no luck
including aerotwitter, relevanced, original play store

**Comment by minyoonji93** [comment]
Same, but I just recently found out it works on the latest timeline, not home recommended timeline for me. 
I've followed this [guide](https://www.reddit.com/r/moddedandroidapps/comments/wt3me0/twifucker_now_supports_video_downloads_in_twitter/) as an rooted device and have all disabled settings on latest timeline (not trending and home timeline tho), download hook and hide settings working. "Deleted _ promoted tweets from app database" at start up always remains 0 too but hopefully this is somewhat helpful.
